bitnami/redis-cluster/templates/redis-statefulset.yaml

```
@@ -234,6 +234,21 @@ spec:
           {{- else if .Values.redis.customReadinessProbe }}
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.redis.customReadinessProbe "context" $) | nindent 12 }}
           {{- end }}
+          {{- if .Values.redis.startupProbe.enabled }}
+          startupProbe:
+            exec:
+              command:
+                - sh
+                - -c
+                - /scripts/ping_liveness_local.sh {{ .Values.redis.startupProbe.timeoutSeconds }}
```

Do we really want to use that complex probe for the "startup"? I'd use a simple _tcpSocket_ probe:
```suggestion
            tcpSocket:
              port: tcp-redis
```